test/models/project_metadata_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
might want to toss in a short description of the file's purpose (or to be purpose) just to keep things consistent and somewhat documented